File: dfhackdays2018ui/src/app/models/student.ts


export class Student {

    constructor(
        public studentId: string,
        public firstName: string,
        public lastName: string,
        public email: string,
        public birthday: Date,
        public gender: string,
        public profession: string,
        public aspirations: string[],
        public signUpDate: Date
    ) { }

}

File: dfhackdays2018api/dfhackdays2018api/Models/Student.cs
using System;
using System.Collections.Generic;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace dfhackdays2018.Models
{
    public class Student
    {
        public Student()
        {
        }

        [BsonId]
        public ObjectId StudentId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public DateTime Birthday { get; set; }
        public string Gender { get; set; }
        public Profession Profession { get; set; }
        public List<Profession> Aspirations { get; set; }
    }
}
